judgments/models.py

```diff
@@ -1,8 +1,54 @@
+import logging
 from collections import defaultdict
+from functools import cached_property
 
+import environ
+import requests
 from django.db import models
 from django.db.models import Max, Min
 
+from judgments.utils import formatted_document_uri
+
+
+class DocumentPdf:
```

Comment:
How much overlap is there with models/utilities/aws.py ? Feels like the internals might more properly belong there, or those functions here.

I think I want this to be a separate file. Happy if we import DocumentPDF here just to provide models.DocumentPDF instead of models.document_pdf.DocumentPDF

Reply:
Cool, I've separated out the models into their own files now